app/Jobs/SendPostToPhoenix.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Rogue\Jobs;
+
+use Rogue\Models\Photo;
+use Rogue\Services\Phoenix;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class SendPostToPhoenix extends Job implements ShouldQueue
+{
+    use InteractsWithQueue;
+
+    /*
+     * Photo instance.
+     *
+     * @var Rogue\Models\Photo
+     */
+    protected $photo;
+
+    /*
+     * @var bool
+     */
+    protected $hasFile;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Photo $photo, $hasFile = true)
```

Comment:
Oh yea we could rename this job. It's really just a temporary thing, we normally won't be sending stuff back to phoenix, just for now until rogue powers the gallery and admin stuff.